servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/RetryingHttpRequesterFilterTest.java

```diff
@@ -171,7 +171,7 @@ public HttpResponse handle(final HttpServiceContext ctx, final HttpRequest reque
                              .build())
                      .appendClientFilter(new Builder()
                              .waitForLoadBalancer(true)
-                             .responseMapper(resp -> resp.status().statusClass().equals(SERVER_ERROR_5XX) ?
+                             .responseMapper(resp -> resp.status().statusClass() == SERVER_ERROR_5XX ?
```

Comment:
ffti: above in [ConnectablePayloadWriter.java](https://github.com/apple/servicetalk/pull/2753/files#diff-4533b6b425f43c7ef436832242a1dcf077631fd7cb800b1f187c55afb91b012e) you used a style something like this, although I don't know that it matters.
```suggestion
                             .responseMapper(resp ->SERVER_ERROR_5XX.equals(resp.status().statusClass()) ?
```

Reply:
Bytecode checkers recommend using `==` for enums (`StatusClass` is enum) and `equals` for objects (`ConnectablePayloadWriter` uses `IOException`)